cordova-lib/src/cordova/plugin.js

```diff
@@ -186,6 +186,14 @@ module.exports = function plugin(command, targets, opts) {
                                 pluginInfoProvider: pluginInfoProvider
                             };
 
+                            if (!('www_dir' in options) || options.www_dir === undefined) {
```

Comment:
IMO it is safe to remove this condition and always set `options.usePlatformWww = true` because this code won't be called by plugman and CLI doesn't know anything about `www_dir` option.